.github/workflows/pull-request.yml

```
@@ -246,7 +312,23 @@
       
       - name: Get changed TypeScript files
         id: changed-files
-        uses: tj-actions/changed-files@v45
+        run: |
+          # Get all changed files
+          ALL_CHANGED_FILES=$(git diff --name-only --diff-filter=ACMRT ${{ github.event.pull_request.base.sha }} ${{ github.event.pull_request.head.sha }})
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)